financeager/resources.py

```diff
@@ -120,3 +122,9 @@ class CopyResource(LogResource):
     def post(self):
         args = copy_parser.parse_args()
         return self.run_safely("copy", error_code=404, **args)
+
+
+class VersionResource(Resource):
```

Comment:
Can you derive the class from `LogResource`. This automatically logs the incoming request which can be helpful for debugging.